core/src/main/resources/defaults/blueprints/cdp-sdx-721.bp

```diff
@@ -222,6 +222,10 @@
           }
         ],
         "serviceConfigs": [
+          {
```

Comment:
Are there unit-tests for this? We should update them.

You have made changes to 7.2.1. Should we make this change in the next version 7.2.2? That way we avoid breaking changes.

Reply:
I am getting familiar with Cloudbreak code. Will find where the test code is and add test code later.

Reply:
This will go to the root disk which is not something we want I think. There are dynamic config providers for HBase [here](https://github.com/hortonworks/cloudbreak/tree/master/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase). 
Configuring the attached disk, here is an [example](https://github.com/hortonworks/cloudbreak/blob/master/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/zookeeper/ZooKeeperVolumeConfigProvider.java#L23) where we configure the first disk only for Zookeper. Here is another example to configure [multiple disks](https://github.com/hortonworks/cloudbreak/blob/master/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hdfs/HdfsVolumeConfigProvider.java#L28) for Hdfs

Reply:
@keyki Hmm.. this goes into HDFS. Can we configure HDFS for different disks?

Reply:
Oh, sorry. I didn't know it's on HDFS. In that case, I'm fine. Thanks for the clarification.